src/foam/dao/OptionDAO.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'OptionDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  documentation: `
```

Comment:
Please improve/expand upon the documentation. I couldn't understand what you meant from reading the documentation and had to read the code figure things out.

Reply:
Ya added more documentation.